parkhangs-expressapp/models/EventCommentModel.js

```diff
@@ -8,7 +8,7 @@ const EventComment = new Schema(
         comment: {type: String, required: true},
         creatorId: {type: Number, required: true},
         creatorName: {type: String, required: true},
-        createdDateTime: {type: Date, required: true}
+        createdDateTime: {type: String, required: true}
```

Comment:
Are you sure you can just change the schema like this?

Reply:
it should be adjusted because Date is inconsistent with all of our other date-related things, which are Strings. That's part of why the created date bug was happening I think.

Reply:
Yeah that was the fix for the  invalid date bug on comments